include/pgduckdb/pg/relations.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "pgduckdb/pg/declarations.hpp"
+
+namespace pgduckdb {
+
+TupleDesc PDRelationGetDescr(Relation relation);
+
+// Not thread-safe. Must be called under a lock.
```

Comment:
It would be nice to actually assert that this lock is held in these functions.

Reply:
This would mean to bring C++ code in this file, which I'm a bit reluctant to do ;-)

Reply:
As discussed offline - will attempt in a separated PR